tools/walletextension/wallet_extension.go

```diff
@@ -74,6 +74,15 @@ func (w *WalletExtension) ProxyEthRequest(request *accountmanager.RPCRequest, co
 	// proxyRequest will find the correct client to proxy the request (or try them all if appropriate)
 	var rpcResp interface{}
 
+	// check if the requested method is GetStorageAt, since we use for gettingUserID if correct parameters are used
```

Comment:
nit: sugestion on comment 
```suggestion
	// wallet extension can override the GetStorageAt to retrieve the current userID
```